libmscore/page.cpp

```diff
@@ -399,9 +407,11 @@ void PageFormat::read(XmlReader& e, Score* score)
                         }
                   }
             else if (tag == "page-height")
-                  _size.rheight() = e.readDouble() * 0.5 / PPI;
+                  _size.rheight() = e.readDouble() / SCALE_XML;
             else if (tag == "page-width")
-                  _size.rwidth() = e.readDouble() * .5 / PPI;
+                  _size.rwidth() = e.readDouble() / SCALE_XML;
+            else if (tag == "page-units")
```

Comment:
I also responded inline in the PR. Your email confused me at first 
because it leaves out the most important line of code, the one below 
your comment, the one immediately below the code you quote here in the 
email:

+ _units = (Ms::Units)e.readInt();

That is the line of code that reads the value as an int (the code you 
are quoting is the read code, not the write code, just to be clear).  
Here is my inline response to your comment:
https://github.com/musescore/MuseScore/pull/2573/files#r61304759

--
Sideways

On 4/27/2016 11:29 AM, Nicolas Froment wrote:
>
> In libmscore/page.cpp 
> <https://github.com/musescore/MuseScore/pull/2573#discussion_r61300930>:
>
> >              else if (tag == "page-width")
> > -                  _size.rwidth() = e.readDouble() * .5 / PPI;
> > +                  _size.rwidth() = e.readDouble() / SCALE_XML;
> > +            else if (tag == "page-units")
>
> I'm not sure it's a good idea to store the unit this way. Maybe it's 
> better to save a string representing the unit. @wschweer 
> <https://github.com/wschweer> Any opinion?
> Also, we should pick a default, probably the unit used in MuseScore 2 
> and not store |page-units| at all if the default is used.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/musescore/MuseScore/pull/2573/files/26336f34193fea4f4e4e6ad052c5624a411478b6#r61300930>
>